src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -390,6 +413,102 @@ def desandboxify_pantsd_loc(path):
       # Update the products with the latest classes.
       self.register_extra_products_from_contexts([ctx.target], compile_contexts)
 
+    def work_for_vts_rsc_jar_library(vts, ctx):
+
+      cp_entries = []
+
+      # Include the current machine's jdk lib jars. This'll blow up remotely.
+      # We need a solution for that.
+      # Probably something to do with https://github.com/pantsbuild/pants/pull/6346
+      # TODO perhaps determine the platform of the jar and use that here.
```

Comment:
This should use @wisechengyi's work from 4d475b37f86f5391504468abddad5eaa619e595d probably.

Reply:
yes. I'd thought I'd broken out an issue for that, but I didn't find one. Filed https://github.com/pantsbuild/pants/issues/6547